packages/components/src/VirtualizedList/CellDatetime/CellDatetime.component.js

```diff
@@ -92,8 +98,10 @@ CellDatetimeComponent.propTypes = {
 	// Column data
 	columnData: PropTypes.shape({
 		tooltipPlacement: PropTypes.string,
+		tooltipLabel: PropTypes.string,
```

Comment:
I dont see why you would like to have a static by props tooltip on a date and because you have the function the need is already cover.

Reply:
Yep I think it's extra, will be removed